toolchain/README.md

```diff
@@ -78,7 +80,27 @@ just by using this toolchain
 > cp ***.tar.gz build/
 ```
 
-Notice: for `CEREAL`, `LibNPY`, `LibRI` and `LibCOMM`, 
+The needed dependencies version default:
+- `cmake` 3.27.6
+- `gcc` 13.2.0 (which will always NOT be installed, But use system)
+- `OpenMPI` 4.1.5
+- `MPICH` 4.1.2
+- `OpenBLAS` 0.3.24 (Intel toolchain need `get_vars.sh` tool from it)
+- `ScaLAPACK` 2.2.1
+- `FFTW` 3.3.10
+- `LibXC` 6.2.2
+- `ELPA` 2023.05.001
+- `CEREAL` 1.3.2
+And Intel-oneAPI need user or server manager to manually install from Intel.
+[Intel-oneAPI](https://www.intel.cn/content/www/cn/zh/developer/tools/oneapi/toolkits.html)
+
+Dependencies below are optional:
+- `LibTorch` 2.0.1
+- `Libnpy` 0.1.0
+- `LibRI` 0.1.0
+- `LibComm` 0.1.0
```

Comment:
Please address the default behavior whether these packages are installed.

Reply:
Done